### ISSUE TITLE
fa-credit-card is different size than id-card and address-card

### ISSUE THREAD

**Issue by kspearrin** [FortAwesome/Font-Awesome#12894]
Hi,
We are migrating from 4.7 to 5 (Pro). We have an implementation in our product where we have a list of items displayed using `fa-ul`.

In 4.7, that list looks something like this:

https://codepen.io/anon/pen/rvNdyQ

```
<ul class="fa-ul">
  <li><i class="fa fa-li fa-globe"></i>Globe</li>
  <li><i class="fa fa-li fa-credit-card"></i>Credit Card</li>
  <li><i class="fa fa-li fa-id-card-o"></i>Id Card</li>
  <li><i class="fa fa-li fa-sticky-note-o"></i>Sticky Note</li>
</ul>
```

![image](https://user-images.githubusercontent.com/1190944/38940619-98e22832-42f8-11e8-8d46-73abb5c3f0e9.png)

In moving to 5, we now have this:

https://codepen.io/anon/pen/zjYWBy

```
<ul class="fa-ul">
  <li><span class="fa-li"><i class="fas fa-globe"></i></span>Globe</li>
  <li><span class="fa-li"><i class="far fa-credit-card"></i></span>Credit Card</li>
  <li><span class="fa-li"><i class="far fa-id-card"></i></span>Id Card</li>
  <li><span class="fa-li"><i class="far fa-sticky-note"></i></span>Sticky Note</li>
</ul>
```

![image](https://user-images.githubusercontent.com/1190944/38940661-baa34cd0-42f8-11e8-8927-7e80b6375251.png)

As you can see, the `fa-id-card` is much smaller than the same shaped `fa-credit-card`. This ends up looking a bit odd when placed in line together like this.

I would like to propose that all "card type" icons be created to the same proportions so that in a situation like the one shown above they would look the same size. I don't know if this means reducing the size of `fa-credit-card`, or increasing the size of other "cards". I would probably suggest that `fa-id-card` and it's related icons be increased to the same size proportions as `fa-credit-card`.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I personally agree, they should look similar

https://fontawesome.com/icons/id-card?style=solid
https://fontawesome.com/icons/credit-card?style=solid

@talbs could you please take a look here?

**Comment by tagliala** [comment]
@kspearrin could you please provide a comprehensive list of affected icons?

I can only find id-card and address-card

**Comment by kspearrin** [comment]
`address-card` also seems affected. Same size as `id-card`.

https://fontawesome.com/icons/address-card?style=solid

**Comment by kspearrin** [comment]
https://fontawesome.com/icons?d=gallery&q=card

**Comment by kspearrin** [comment]
Any update? The status of this issue is not correct. It should not be "waiting for feedback" any longer.

**Comment by tagliala** [comment]
@kspearrin thanks

I think that closed captioning should follow the same size but I don't know if it is smaller by purpose

@talbs what do you think?

**Comment by tagliala** [comment]
Should https://fontawesome.com/icons/barcode-alt?style=solid and https://fontawesome.com/icons/address-card?style=solid have the same size?

If there is a grid, they should

**Comment by tagliala** [comment]
I can confirm this ()fa-credit-card is different size than id-card and address-card) has been fixed in 5.0.11

Feel free to start another issue about the other question
